core/presentation/src/main/java/com/nibalk/tasky/core/presentation/components/TaskyPhotoCarousel.kt

```diff
@@ -0,0 +1,213 @@
+package com.nibalk.tasky.core.presentation.components
+
+import androidx.compose.foundation.border
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.sharp.Add
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import com.nibalk.tasky.core.presentation.R
+import com.nibalk.tasky.core.presentation.themes.LocalSpacing
+import com.nibalk.tasky.core.presentation.themes.TaskyDarkGray
+import com.nibalk.tasky.core.presentation.themes.TaskyGray
+import com.nibalk.tasky.core.presentation.themes.TaskyTheme
+
+@Composable
+fun TaskyPhotoCarousel(
+    maxPhotoCount: Int = 10,
+    photos: List<ByteArray?> = emptyList(),
+    isEditable: Boolean,
+    onPhotoClicked: (ByteArray?) -> Unit,
+    onPhotoAddIconClicked: () -> Unit,
+) {
+    val spacing = LocalSpacing.current
+    val arePhotosFull = false
```

Comment:
I don't see how this is used inside this function. You use it once to check whether to show the add photo icon, but it's never switched to true or am I wrong?